01-Lab/SoftwareIntro.ipynb

```diff
@@ -4,253 +4,459 @@
+    "\n",
+    "Download from osf: https://osf.io/bprq5/\n",
     "\n",
-    "### Load python package\n",
-    "The following python package is necessary to view the nifti images that we will be working with in this tutorial."
+    "After clicking on the link, you should see the download __lab01_images.tar.gz__ appear in your browser downloads. We will practice using the bash terminal to navigate to the Downloads folder, and to copy and unpack the download to our lab directory.\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "from niwidgets import NiftiWidget"
+    "%%bash\n",
+    "#^the %% symbols tell the notebook to execute bash commands\n",
+    "cd ~/Downloads  #change directories to the Downloads folder, the ~ is shorthand for \"home directory\"\n",
+    "ls  #list the files there"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "raw",
    "metadata": {},
    "source": [
-    "### Download the data\n",
-    "We need one more python package to download publically available data for viewing, and we will download one subject for playing with."
+    "%%bash\n",
+    "pwd  #tell me where I am again?\n",
+    "#when I move cells, I moved back to the 01-Lab directory,but I can still move the file to here\n",
+    "mv /home/brain/Downloads/lab01_images.tar.gz ./lab01_images.tar.gz"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "%%bash\n",
+    "ls #is it here?\n",
+    "#unpack a tar file: tar -zxvf tar-archive-name.tar.gz\n",
+    "tar -zxvf lab01_images.tar.gz"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "ls  #now osfshare is a directory, we will go into this directory to practice opening MR images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "from nilearn import datasets\n",
-    "haxby_dataset = datasets.fetch_haxby()\n",
-    "func_filename = haxby_dataset.func[0]\n",
-    "anat_filename = haxby_dataset.anat[0]\n",
+    "%%bash\n",
+    "cd osfshare\n",
+    "ls  #list contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Preview the data directly in jupyter\n",
     "\n",
-    "# MV\n",
-    "# note download of the last file takes a really long time with a slow-ish connection\n",
-    "# consider getting most if not all of data needed for labs loaded in to the machine before they download it?\n"
+    "We'll first get used to terms about the spatial orientation of an MR image. We'll use a viewer that is part of nipype, which is a collection of neuroimaging tools written in the python programming language. \n",
```

Comment:
[nipy](http://nipy.org/) is the "parent" organization and niwidgets/nipype are some of the projects that offshoot from nipy.